framework/include/meshgenerators/SideSetsFromBoundingBoxGenerator.h

```diff
@@ -12,18 +12,14 @@
 
 #include "MeshGenerator.h"
 #include "MooseEnum.h"
+#include "libmesh/bounding_box.h"
```

Comment:
Why undo forward declaration?

Reply:
The class actually holds an instance of a `BoundingBox`, not just a reference or a pointer. A forward declaration is not sufficient. Previously the `bounding_box` header was getting pulled in from some other location...that didn't actually need the header